src/test/java/com/aws/greengrass/logmanager/CloudWatchAttemptLogsProcessorTest.java

```diff
@@ -221,17 +250,26 @@ public void GIVEN_one_component_one_file_less_than_max_WHEN_locate_throws_Servic
         assertTrue(attempt.getLogStreamsToLogEventsMap().containsKey(logStream));
         CloudWatchAttemptLogInformation logEventsForStream1 = attempt.getLogStreamsToLogEventsMap().get(logStream);
         assertNotNull(logEventsForStream1.getLogEvents());
-        assertEquals(7, logEventsForStream1.getLogEvents().size());
+        assertEquals(13, logEventsForStream1.getLogEvents().size());
         assertTrue(logEventsForStream1.getAttemptLogFileInformationMap().containsKey(file1.getAbsolutePath()));
         assertEquals(0, logEventsForStream1.getAttemptLogFileInformationMap().get(file1.getAbsolutePath()).getStartPosition());
-        assertEquals(13, logEventsForStream1.getAttemptLogFileInformationMap().get(file1.getAbsolutePath()).getBytesRead());
+        assertEquals(2943, logEventsForStream1.getAttemptLogFileInformationMap().get(file1.getAbsolutePath()).getBytesRead());
```

Comment:
Nit: consider refactoring these test cases so you don't need to make all these edits every time a test file changes